#!/usr/bin/env python
PACKAGE = "air3d_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("radius",    double_t,    0, "Radius of the outer", 2.75,  0, 10.0)
gen.add("delta_radius",    double_t,    0, "For intermediate drone radius", 1.0, 0.5, 2.0)
gen.add("height",   double_t,    0, "Height of the outer", 2.0,  -10.0, 10.0)
gen.add("delta_height",   double_t,    0, "For intermediate height", 0.0,  -10.0, 10.0)
gen.add("angle",   double_t,    0, "Angle of the outer", 0.0,  -180., 180.0)
gen.add("delta_angle",   double_t,    0, "Angle of the inner wrt outer", 0.0, -180., 180.0)
gen.add("USE_CATENARY", bool_t, 0, "Use catenary feed-forward",  True)
gen.add("do_task", bool_t, 0, "Use catenary feed-forward",  False)

exit(gen.generate(PACKAGE, "air3d_planner", "MissionControl"))